chapters/7-publication.tex

```diff
@@ -338,7 +338,7 @@ \subsection{De-identifying data for publication}
 such as a computational reproducibility check required for publication,
 if done under careful data security protocols and approved by an IRB.
 
-\subsection{Publishing clean and constructed datasets}
+\subsection{Publishing analysis datasets}
```

Comment:
```suggestion
\subsection{Publishing research datasets}
```
I think we talk both about the final analysis datasets and the clean data when it's self-collected?